src/components/guidance-content/info/InformationItems.js

```diff
@@ -0,0 +1,617 @@
+    <Button bsSize='small'><Glyphicon glyph='save-file'/></Button> {' '}
+    (Polygon Import/Export)
+  </span>
+);
+
+const mapColourScaleControls = (
+  <span>
+    (Colour Scale Bar)
+  </span>
+);
+
+const mapAutoScaleControl = (
+  <span>
+    <Button bsSize='small'>
+      <span style={{ fontWeight: 'bold' }}>AS</span>
+    </Button> {' '}
+    (Auto-Scale)
+  </span>
+);
+
+export const mapPanelLabel = (
+  <LabelWithInfo label='Data Map'>
+     <p>
+       Map displaying data selected by
+       Model, Emission Scenario, and Variable(s).
+     </p>
+    <p>
+      Summary of map tools and other controls.
+      (For details, see <Link to='/help/general'>Help</Link>.)
+    </p>
+    <ul className={css.controlsList}>
+      <li>
+        {mapZoomControls}: Zoom map in and out.
+      </li>
+      <li>
+        {mapPolygonDrawControls}: Draw polygons on the map.
+        Polygons determine the extents over which spatial data averaging is
+        performed.
+      </li>
+      <li>
+        {mapPolygonEditControls}: Edit and delete polygons on the map.
+        Polygons determine the extents over which spatial data averaging is
+        performed.
+      </li>
+      <li>
+        {mapPolygonImportExportControls}: Import and export polygons on the map.
+        Polygons determine the extents over which spatial data averaging is
+        performed.
+      </li>
+      <li>
+        {mapSettingsControl}: Select which dataset(s) are displayed and how.
+      </li>
+      <li>
+        {mapColourScaleControls}: Displays data value ⇄ colour mapping.
+      </li>
+      <li>
+        {mapAutoScaleControl}: Sets bounds of data value ⇄ colour mapping to
+        current range of data.
+      </li>
+    </ul>
+  </LabelWithInfo>
+);
+
+// The following items are rendered in the Map Settings dialog.
+// Using <Link/> components in the body cause the info popup not to appear.
+// Specific cause unknown. Avoid them in these items.
+
+export const TimeSelectorLabel = ({ temporalLabelPart }) => (
+  <LabelWithInfo label={temporalLabelPart}>
+    <p>
+      Select the portion of the year over which data values are averaged
+      before those portion averages are averaged over a multi-decadal period.
+    </p>
+  </LabelWithInfo>
+);
+
+export const paletteSelectorLabel = (
+  <LabelWithInfo label={`${name} Colour Palette`}>
+    <p>
+      Select the set of colours to which data values are mapped
+      for representation on the map.
+    </p>
+  </LabelWithInfo>
+);
+
+export const colourMapTypeSelectorLabel = (
+  <LabelWithInfo label='Colour Map Type'>
+    <p>
+      Select how data values are mapped to colours.
+      For details on linear and logarithmic mappings,
+      see Help.
+    </p>
+  </LabelWithInfo>
+);
+
+///////////////////////////////
+// Graphs
+///////////////////////////////
+
+const annualCycleGraphDefn = `
+  An annual cycle graph presents spatially averaged values of a multi-year 
```

Comment:
I wonder if the first "An" should be a "The" or "This"  - is what we're saying here true of all annual cycle graphs in general, or just the ones in CE?

Reply:
Even if it's just the ones in CE, we present several. I don't know whether there is a general notion of annual cycle graph in common use, but I intended this to describe our coinage.